styles/styles.css

```
@@ -0,0 +1,305 @@
+@font-face {
+    font-family: "PT Sans";
+    font-style: normal;
+    font-weight: 400;
+    src: url("../fonts/ptsans-400.woff2") format("woff2");
+    font-display: swap;
+  }
+  
+  @font-face {
+    font-family: "PT Sans";
+    font-style: normal;
+    font-weight: 700;
+    src: url("../fonts/ptsans-700.woff2") format("woff2");
+    font-display: swap;
+  }
+  body {
+    font-family: "PT Sans", sans-serif;
+    font-size: 18px;
+    line-height: 21px;
+    color: #333333;
+    background-color: #ffffff;
+  }
+  .button {
+    font-family: inherit;
+    font-size: 20px;
+    line-height: 26px;
+    font-weight: 700;
+    text-align: center;
+    color: #ffffff;
+    background-color: #83B3D3;
+    text-decoration: none;
+    text-transform: uppercase;
+  }
```

Между селекторами нужно добавлять отступ
```suggestion
  body {
    font-family: "PT Sans", sans-serif;
    font-size: 18px;
    line-height: 21px;
    color: #333333;
    background-color: #ffffff;
  }
  
  .button {
    font-family: inherit;
    font-size: 20px;
    line-height: 26px;
    font-weight: 700;
    text-align: center;
    color: #ffffff;
    background-color: #83B3D3;
    text-decoration: none;
    text-transform: uppercase;
  }
```